pages/api/website/[id]/metrics.js

```diff
@@ -41,6 +41,8 @@ export default async (req, res) => {
 
     if (sessionColumns.includes(type)) {
       let data = await getSessionMetrics(websiteId, startDate, endDate, type, {
+        url,
+        referrer,
```

Comment:
We don't need to do referrer session metrics.

Reply:
Shouldn't we get session metrics on a specific page or from a referrer? Like, how many macOS users visited `/product/macos` and how many users from `github.com` use Linux?
I'm not sure if that metrics are useful in web analytics. I'm starting to get familiar with analytics 🤔